dashboard/app/models/puzzle_rating.rb

```diff
@@ -28,7 +28,7 @@ class PuzzleRating < ActiveRecord::Base
   validates_uniqueness_of :user_id, :scope => [:script_id, :level_id], :allow_nil => true
 
   def PuzzleRating.enabled?
-    true
+    false
```

Comment:
It removes the UI as well; I'm pretty opposed to having intentionally non-functional UI components.